### PR TITLE
Column change

### PR DESCRIPTION
**Pull request by Dtphelan1**
Straightforward. Updates the column names for our CSV format, places where those properties are used for accessing the values extracted from our CSVModule, and any documentation/fixtures related to that format.

### CODE REVIEW

File: test/extractors/CSVCancerDiseaseStatusExtractor.test.js [hunk 1]
```diff
@@ -34,35 +34,35 @@ test('CSVCancerDiseaseStatusExtractor -> joinAndReformatData', () => {
   // Test all required properties are
   delete localData[0].mrn;
   expect(() => joinAndReformatData(localData)).toThrow(
-    new Error('DiseaseStatusData missing an expected property: mrn, subjectFHIRId, conditionFHIRId, diseaseStatus and dateOfObservation are required.'),
+    new Error('DiseaseStatusData missing an expected property: mrn, subjectId, conditionFHIRId, diseaseStatus and dateOfObservation are required.'),
```
Comment (mgramigna) [inline test/extractors/CSVCancerDiseaseStatusExtractor.test.js:37]:
```suggestion
    new Error('DiseaseStatusData missing an expected property: mrn, subjectId, conditionId, diseaseStatus and dateOfObservation are required.'),
```

File: src/extractors/CSVCancerDiseaseStatusExtractor.js [hunk 2]
```diff
@@ -8,8 +8,8 @@ function joinAndReformatData(arrOfDiseaseStatusData) {
   // No join needed - just reformatting for template
   // Check the shape of the data
   arrOfDiseaseStatusData.forEach((record) => {
-    if (!(record.mrn && record.subjectFHIRId && record.conditionFHIRId && record.diseaseStatus && record.dateOfObservation)) {
-      throw new Error('DiseaseStatusData missing an expected property: mrn, subjectFHIRId, conditionFHIRId, diseaseStatus and dateOfObservation are required.');
+    if (!(record.mrn && record.subjectId && record.conditionFHIRId && record.diseaseStatus && record.dateOfObservation)) {
+      throw new Error('DiseaseStatusData missing an expected property: mrn, subjectId, conditionFHIRId, diseaseStatus and dateOfObservation are required.');
```
Comment (mgramigna) [inline src/extractors/CSVCancerDiseaseStatusExtractor.js:12]:
```suggestion
    if (!(record.mrn && record.subjectId && record.conditionId && record.diseaseStatus && record.dateOfObservation)) {
      throw new Error('DiseaseStatusData missing an expected property: mrn, subjectId, conditionId, diseaseStatus and dateOfObservation are required.');
```

File: src/extractors/CSVCancerDiseaseStatusExtractor.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ function joinAndReformatData(arrOfDiseaseStatusData) {
       display: record.diseaseStatus,
     },
     subject: {
-      id: record.subjectFHIRId,
+      id: record.subjectId,
     },
     condition: {
       id: record.conditionFHIRId,
```
Comment (mgramigna) [inline src/extractors/CSVCancerDiseaseStatusExtractor.js:27]:
```suggestion
      id: record.conditionId,
```

File: test/extractors/CSVCancerDiseaseStatusExtractor.test.js [hunk 4]
```diff
@@ -34,35 +34,35 @@ test('CSVCancerDiseaseStatusExtractor -> joinAndReformatData', () => {
...
   );
   localData[0].mrn = exampleCSVDiseaseStatusModuleResponse[0].mrn;
   expect(joinAndReformatData(localData)).toEqual(expect.anything());
 
-  delete localData[0].subjectFHIRId;
+  delete localData[0].subjectId;
   expect(() => joinAndReformatData(localData)).toThrow(
-    new Error('DiseaseStatusData missing an expected property: mrn, subjectFHIRId, conditionFHIRId, diseaseStatus and dateOfObservation are required.'),
+    new Error('DiseaseStatusData missing an expected property: mrn, subjectId, conditionFHIRId, diseaseStatus and dateOfObservation are required.'),
```
Comment (mgramigna) [inline test/extractors/CSVCancerDiseaseStatusExtractor.test.js:44]:
```suggestion
    new Error('DiseaseStatusData missing an expected property: mrn, subjectId, conditionId, diseaseStatus and dateOfObservation are required.'),
```

File: test/extractors/CSVCancerDiseaseStatusExtractor.test.js [hunk 5]
```diff
@@ -34,35 +34,35 @@ test('CSVCancerDiseaseStatusExtractor -> joinAndReformatData', () => {
...
   );
-  localData[0].subjectFHIRId = exampleCSVDiseaseStatusModuleResponse[0].subjectFHIRId;
+  localData[0].subjectId = exampleCSVDiseaseStatusModuleResponse[0].subjectId;
   expect(joinAndReformatData(localData)).toEqual(expect.anything());
 
   delete localData[0].conditionFHIRId;
```
Comment (mgramigna) [inline test/extractors/CSVCancerDiseaseStatusExtractor.test.js:49]:
```suggestion
  delete localData[0].conditionId;
```

File: test/extractors/CSVCancerDiseaseStatusExtractor.test.js [hunk 6]
```diff
@@ -34,35 +34,35 @@ test('CSVCancerDiseaseStatusExtractor -> joinAndReformatData', () => {
...
   expect(() => joinAndReformatData(localData)).toThrow(
-    new Error('DiseaseStatusData missing an expected property: mrn, subjectFHIRId, conditionFHIRId, diseaseStatus and dateOfObservation are required.'),
+    new Error('DiseaseStatusData missing an expected property: mrn, subjectId, conditionFHIRId, diseaseStatus and dateOfObservation are required.'),
```
Comment (mgramigna) [inline test/extractors/CSVCancerDiseaseStatusExtractor.test.js:51]:
```suggestion
    new Error('DiseaseStatusData missing an expected property: mrn, subjectId, conditionId, diseaseStatus and dateOfObservation are required.'),
```

File: test/extractors/CSVCancerDiseaseStatusExtractor.test.js [hunk 7]
```diff
@@ -34,35 +34,35 @@ test('CSVCancerDiseaseStatusExtractor -> joinAndReformatData', () => {
...
   );
   localData[0].conditionFHIRId = exampleCSVDiseaseStatusModuleResponse[0].conditionFHIRId;
```
Comment (mgramigna) [inline test/extractors/CSVCancerDiseaseStatusExtractor.test.js:53]:
```suggestion
  localData[0].conditionId = exampleCSVDiseaseStatusModuleResponse[0].conditionId;
```

File: test/extractors/CSVCancerDiseaseStatusExtractor.test.js [hunk 8]
```diff
@@ -34,35 +34,35 @@ test('CSVCancerDiseaseStatusExtractor -> joinAndReformatData', () => {
...
   expect(joinAndReformatData(localData)).toEqual(expect.anything());
 
   delete localData[0].diseaseStatus;
   expect(() => joinAndReformatData(localData)).toThrow(
-    new Error('DiseaseStatusData missing an expected property: mrn, subjectFHIRId, conditionFHIRId, diseaseStatus and dateOfObservation are required.'),
+    new Error('DiseaseStatusData missing an expected property: mrn, subjectId, conditionFHIRId, diseaseStatus and dateOfObservation are required.'),
```
Comment (mgramigna) [inline test/extractors/CSVCancerDiseaseStatusExtractor.test.js:58]:
```suggestion
    new Error('DiseaseStatusData missing an expected property: mrn, subjectId, conditionId, diseaseStatus and dateOfObservation are required.'),
```

File: test/extractors/CSVCancerDiseaseStatusExtractor.test.js [hunk 9]
```diff
@@ -34,35 +34,35 @@ test('CSVCancerDiseaseStatusExtractor -> joinAndReformatData', () => {
...
   );
   localData[0].diseaseStatus = exampleCSVDiseaseStatusModuleResponse[0].diseaseStatus;
   expect(joinAndReformatData(localData)).toEqual(expect.anything());
 
   delete localData[0].dateOfObservation;
   expect(() => joinAndReformatData(localData)).toThrow(
-    new Error('DiseaseStatusData missing an expected property: mrn, subjectFHIRId, conditionFHIRId, diseaseStatus and dateOfObservation are required.'),
+    new Error('DiseaseStatusData missing an expected property: mrn, subjectId, conditionFHIRId, diseaseStatus and dateOfObservation are required.'),
```
Comment (mgramigna) [inline test/extractors/CSVCancerDiseaseStatusExtractor.test.js:65]:
```suggestion
    new Error('DiseaseStatusData missing an expected property: mrn, subjectId, conditionId, diseaseStatus and dateOfObservation are required.'),
```

File: test/extractors/fixtures/csv-cancer-disease-status-module-response.json [hunk 10]
```diff
@@ -1,7 +1,7 @@
 [
   {
     "mrn": "pat-mrn-1",
-    "subjectFHIRId": "pat-fhir-1",
+    "subjectId": "subjectId-1",
     "conditionFHIRId": "cond-1",
```
Comment (mgramigna) [inline test/extractors/fixtures/csv-cancer-disease-status-module-response.json:5]:
```suggestion
    "conditionId": "cond-1",
```

**Comment by Dtphelan1** [comment]
Good catch - missed that column change in my gitdiff checks.

**Pull request merged**